### DISCUSSION TITLE
Feat: Add support to disable the "Pretty"-function in a trace

### DISCUSSION THREAD

**Discussion by Cornelius000** [langfuse/langfuse#4790] [Ideas]
### Describe the feature or potential improvement

We have some traces where the data is not being displayed that good because of the pretty function. 

Maybe add a toggle in the project settings where we can choose to disable the toggle and only dislpay the trace data as JSON.

**Comment by marcklingen** [comment]
Makes sense, thanks for sharing.

Could you share a screenshot or public link of how this looks like? "Pretty" should not cause UI glitches and I'd love to fix this
